Makefile

```diff
@@ -238,6 +238,35 @@ create-cluster-ha: binaries-dev ## Create a development Kubernetes cluster on AW
 	-p ./cmd/clusterctl/examples/aws/out/provider-components.yaml \
 	-a ./cmd/clusterctl/examples/aws/out/addons.yaml
 
+.PHONY: create-cluster-ha-management
+create-cluster-ha-management: ## Create a development Kubernetes cluster on AWS using HA examples in a KIND management cluster.
```

Comment:
afaict this only creates a single control plane machine and not HA as this example says.

Reply:
True, need to fix it :D

Reply:
Fixed!